src/5e-SRD-Equipment-Categories.json

```diff
@@ -2280,5 +2232,72 @@
 			}
 		],
 		"url": "/api/equipment-categories/waterborne-vehicles"
+	},
+	{
+		"index": "arcane-foci",
```

Comment:
For these three, should we update the corresponding equipment to point to this instead of Adventuring Gear? Or update the `gear category`?

Reply:
Good spot. The `gear_category` should updated to be plural, to match the equipment categories' names.

The `equipment_category` of the various foci should remain "Adventuring Gear", I think, to match the way it's done in other types of adventuring gear that have their own equipment sub-categories (e.g. kits, artisan's tools, vehicles).

Perhaps a good PR for the future would swap the `gear_category` string for a reference to the equipment sub-category? For now, I'll switch gear categories to plural names.

Reply:
I think that would be a good future PR!

Reply:
Sorted.